content/en/docs/external-add-ons/kserve/migration.md

```diff
@@ -0,0 +1,89 @@
+# Migrating from KFServing
+
+### **KFServing is now KServe**
+###[Blog Post](https://kserve.github.io/website/0.7/blog/articles/2021-09-27-kfserving-transition/)
+
+We are excited to announce the next chapter for KFServing.
+In coordination with the Kubeflow Project Steering Group, the [<u>KFServing GitHub repository</u>](https://github.com/kubeflow/kfserving) has now been
+transferred to an independent [<u>KServe GitHub organization</u>](https://github.com/kserve/kserve) under the stewardship of the Kubeflow Serving Working
+Group leads.
+
+
+This doc explains how to migrate existing inference services from KFServing to KServe without downtime.
```

Comment:
nit: Change "inference services" to "InferenceServices" to stay consistent with the CRD name.